src/server/controllers/zitadel.ts

```diff
@@ -0,0 +1,23 @@
+import type {Request, Response} from '@gravity-ui/expresskit';
+
+import {appHostUri, clientId, zitadelUri} from '../app-env';
+
+export async function logout(req: Request, res: Response) {
+    if (!clientId) {
+        throw new Error('Missing CLIENT_ID in env');
```

Comment:
I prefer errors rather than exceptions, but not critical, since there are exceptions below